index.html

```diff
@@ -97,9 +97,50 @@
     ;;
 esac
 
-echo "[+] Downloading Qovery CLI archive..."
 
-latest_tag=$(curl --silent "https://api.github.com/repos/$repo/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+find_latest_tag() {
+  # Find through API first
+  echo "  [+] Trying through Github API" >&2
+  response=$(curl --silent --write-out "HTTPSTATUS:%{http_code}" "https://api.github.com/repos/$repo/releases/latest" 2>/dev/null || echo "HTTPSTATUS:000")
+  http_code=$(echo "$response" | grep -o "HTTPSTATUS:[0-9]*" | cut -d: -f2)
+  body=$(echo "$response" | sed -E 's/HTTPSTATUS:[0-9]*$//')
+
+  if [ "$http_code" -eq 200 ] && [ -n "$body" ]; then
+    tag=$(echo "$body" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' 2>/dev/null)
+    if [ -n "$tag" ] && [ "$tag" != "null" ]; then
+      echo "  [+] Successfully retrieved tag from GitHub API: $tag" >&2
+      echo "$tag"
+      return
```

Comment:
Without the return (here + line 130), the program continued, leading to the return version to be inconsistent (2 lines where both method to retrieve the tag worked)
```
1.40.5
v1.40.5
```